src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts

```diff
@@ -28,7 +27,7 @@ export class ManageAuthorisedPeopleHandler extends GenericHandler {
         const lang = getTranslationsForView(req.t, constants.MANAGE_AUTHORISED_PEOPLE_PAGE);
         this.viewData = this.getViewData(companyNumber, lang);
         const cancellation: Cancellation = getExtraData(req.session, constants.CANCEL_PERSON);
-        let companyAssociations: Associations = await getCompanyAssociations(req, companyNumber, undefined, undefined, pageNumber - 1);
+        let companyAssociations: AssociationList = await getCompanyAssociations(req, companyNumber);
```

Comment:
Is pageNumber no longer required for getting company associations? How pagination is going to work without this?

Reply:
Yeah, this looks incorrect. I will update.